.github/workflows/windows.yml

```diff
@@ -31,15 +29,18 @@ jobs:
       contents: write
     strategy:
       matrix:
+        secure: ['', 'tls']
+        cc: ['gcc', 'msvc']
         include:
-          - compiler: GCC
+          - cc: gcc
             make: mingw32-make
-            tls_include: "C:\\Program Files\\OpenSSL\\include"
-            tls_lib: "C:\\Program Files\\OpenSSL\\lib"
-          - compiler: MSVC
+          - cc: gcc
+            make: mingw32-make
+          - cc: msvc
             make: nmake
-            tls_include: "C:\\Program Files\\OpenSSL\\include"
-            tls_lib: "C:\\Program Files\\OpenSSL\\lib"
+        exclude:
+          - cc: msvc
+            secure: tls
```

Comment:
> Sure, I can't see the screenshot as GitHub servers recently has traffic issues... however, I'm working on webui\src\webview/WebView2.h(62): error C2061: syntax error: identifier 'ICoreWebView2AcceleratorKeyPressedEventArgs'.

Prepped it in a way that when the fix is finished you only would need to remove this exclude then also the msvc tls builds will be added.

Line 33 can be removed then too, it is obsolete without the `exclude`.

In case you don't see the screenshot, it shows the artifacts in the workflow summary, including the uploaded tls artifact. Also added a link to the summary to directly check it in the browser.